components/brave_extension/extension/brave_extension/background/events/cosmeticFilterEvents.ts

```diff
@@ -47,7 +47,6 @@ chrome.contextMenus.create({
   parentId: 'brave',
   contexts: ['all']
 })
```

Comment:
You still have all 3 of the menu options registered here, even though the first does nothing anymore

Reply:
fixed